### DISCUSSION TITLE
Creating custom table insertion with specific class

### DISCUSSION THREAD

**Discussion by RokasDie** [ueberdosis/tiptap#969] [General Questions]
Hello,

I want to ask, how can I make a custom table insertion. I need to have something like this:

- "Infobox" which would float to the right of the content with max-width of 300px 
- This would be a separate table from the normal table.

I am having difficulty understanding where I would be adding the custom class, so that inserted custom table would have it. Do I need to go to Prosemirror code for that?

I would really appreciate some help or directions

**Comment by mikbox74** [comment]
Hello @RokasDie ! I have same task and I tried to extend my custom Table plugin from same built-in one but unsuccessful.
[Here I described my attempt](https://github.com/ueberdosis/tiptap/discussions/992). Write please if you will have an idea how to push it forward
